modules/storage-api/src/main/java/org/apache/ignite/internal/storage/TableRowAndRowId.java

```diff
@@ -24,7 +24,7 @@
  * Wrapper that holds both {@link TableRow} and {@link RowId}. {@link TableRow} is null for tombstones.
  */
 public class TableRowAndRowId {
-    /** Table row. */
+    /** Table row. {@code null} if tombstone. */
```

Comment:
Now this is not entirely true, we discussed that this is possible even not for tombstones)
I propose to describe this in the documentation for the method.